static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js

```diff
@@ -1,41 +1,69 @@
 // @flow
-import React, { Component } from 'preact-compat';
+import React from 'preact-compat';
 import config from 'lib/config';
 import { FollowCard } from 'common/modules/identity/upsell/consent-card/FollowCard';
+import {
+    setConsentsInApi,
+    getNewsLetterConsents,
+    getUserConsents,
+} from 'common/modules/identity/upsell/store/consents';
 import type { ConsentWithState } from '../store/types';
-import { setConsentsInApi } from '../store/consents';
 import { ErrorBar, genericErrorStr } from '../error-bar/ErrorBar';
 import { ExpanderButton } from '../button/ExpanderButton';
 
-type FollowCardListProps = {
-    consents: Promise<ConsentWithState>[],
+const getConsents = (): Promise<ConsentWithState[]> => {
+    const userConsents = getUserConsents([
+        'supporter',
+        'holidays',
+        'events',
+        'offers',
+        'jobs',
+    ]);
+
+    const newsLetterConsents = getNewsLetterConsents([
+        'today-uk',
+        'the-long-read',
+        'bookmarks',
+        'brexit-briefing',
+        'green-light',
+        'lab-notes',
+    ]);
+
+    return Promise.all([userConsents, newsLetterConsents]).then(consents =>
```

Comment:
massive personal nitpick but i feel this would be clearer
```suggestion
    return Promise.all([userConsents, newsLetterConsents])
        .then(([fetchedUserConsents,fetchedNewsLetterConsents]) =>
            [...fetchedUserConsents, ...fetchedNewsLetterConsents]
        )
```

Reply:
nice 😎